app/models/study_topic.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: study_topics
+#
+#  id          :integer          not null, primary key
+#  name        :text             not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  description :text
+#
+# Indexes
+#
+#  index_study_topics_on_name  (name) UNIQUE
+#
+
+class StudyTopic < ActiveRecord::Base
+  has_many :studies, inverse_of: :study_topic
+
+  validates_presence_of :name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.